### DISCUSSION TITLE
How to check which distrib I am using?

### DISCUSSION THREAD

**Discussion by runinrunin** [MiKTeX/miktex#1524] [Q&A]
Hello, 
I am new with latex. I read some docs and decide to start with Miktex. I will use vscode as editor. But there is one thing that annoyed me: How can I check which distrib is used by my system and how can I change it?

My system is macos: I though to use the "which" command in the terminal. "which latex" give me $/Library/TeX/texbin/latex.
Typing "latex" give me &This is pdfTeX, Version 3.141592653-2.6-1.40.26 (TeX Live 2024) (preloaded format=latex)

Thanks.

**Comment by LLdaniel** [comment]
Seems you already have your answer in your question :sweat_smile: 
> Typing "latex" give me &This is pdfTeX, Version 3.141592653-2.6-1.40.26 (TeX Live 2024) (preloaded format=latex)

Seems your currently used `pdflatex` compiler is shipped with TexLive 2024 distribution not MikTeX.

Maybe this helps when switching to MiKTeX: https://miktex.org/howto/install-miktex-mac
